src/debugger.ts

```diff
@@ -174,14 +172,6 @@ export namespace Debugger {
       return this._modeChanged;
     }
 
-    get codeValue() {
-      return this._codeValue;
-    }
-
-    set codeValue(observableString: IObservableString) {
-      this._codeValue = observableString;
-    }
-
```

Comment:
Nice! It's been a while that I wanted to remove this from the model, I'm glad you did it!